api/src/api/notes.js

```diff
@@ -21,20 +21,21 @@ const { encryptNote, decryptNote } = require('../utils/notes');
  * @returns {Array<Object>}
  */
 const memberFromId = async (ids) => {
-  const memberPromises = ids
-    // filters out invalid ids
-    .filter((id) => !!id)
-    // creates promises for each valid id
-    .map((memberId) => Member.findById(memberId));
-
-  // get member data
-  const members = await Promise.all(memberPromises);
+  const members = await Member.find(
+    { _id: { $in: ids } },
+    { firstName: 1, lastName: 1 },
+  );
 
+  const mappedMembers = {};
   // return derived full name and id from meber
-  return members.map((member) => ({
-    memberId: member._id,
-    name: `${member.firstName} ${member.lastName}`,
-  }));
+  members.forEach(
```

Comment:
could we directly assign `mappedMembers` with `members.map( ... )` instead of using a side-effect here?